zk_prover/src/chips/overflow/overflow_check.rs

```diff
@@ -98,20 +100,23 @@ impl<const MAX_BITS: u8, const MOD_BITS: usize> OverflowChip<MAX_BITS, MOD_BITS>
 
         meta.annotate_lookup_any_column(range, || "LOOKUP_MAXBITS_RANGE");
 
-        for i in 0..num_rows {
-            meta.lookup_any("range check for MAXBITS", |meta| {
-                let cell = meta.query_advice(b, Rotation(i as i32));
-                let range = meta.query_fixed(range, Rotation::cur());
-                let enable_lookup = meta.query_selector(toggle_overflow_check);
-                vec![(enable_lookup * cell, range)]
-            });
-        }
+        meta.lookup_any("range check for MAXBITS", |meta| {
+            let cell = meta.query_advice(b, Rotation::cur());
+            let range = meta.query_fixed(range, Rotation::cur());
+
+            // print range
+            println!("range: {:?}", range);
```

Comment:
Did this line of code possibly remain here by mistake after testing?

Reply:
Yep 🥲 fixed at f029de